### ISSUE TITLE
Plurals in the tag-tanslation files supported?

### ISSUE THREAD

**Issue by Hufkratzer** [plepe/openstreetmap-tag-translations#45]
[README.md](https://github.com/plepe/openstreetmap-tag-translations/blob/master/README.md) explains that plurals can/should be defined with "!=1": like:
> {
>     "tag:barrier": {
>         "message": "Barrière",
>         "!=1": "Barrières"
>     },
> ...

Is this still correct? Currently [en.json](https://github.com/plepe/openstreetmap-tag-translations/blob/master/tags/en.json) doesn't contain any plural definitions, some plurals have been [removed here](https://github.com/plepe/openstreetmap-tag-translations/commit/e6eaf1595e1284e5cb35d5e2cc5783b2db20343c#diff-4372cd89eec6b6e9abedef58abc38963) by @Cybereric . Why have they been removed?

I think it would be useful to have them when needed. For example the routes categories contain a hard coded string "Routes". If we had defined 
>     "tag:type=route": {
>         "message": "Route",
>         "!=1": "Routes"
>     }, ...

we could use it there. Also in filters plurals would make sense (e.g. "Hotels" instead of "Hotel").

How could plurals be retrieved by the translation functions, e.g. what to use instead of tagTrans('type', "route") to get "Routes" instead of "Route"?

**Comment by plepe** [comment]
No, they are no longer supported. The Weblate application does not support this (at least in this format) and they were removed by it. I'm not too happy about this too. When I would add support for this, I would recover the already existing plurals from the Git history.
